server/utils/brew.js

```diff
@@ -0,0 +1,41 @@
+const yaml = require('js-yaml');
+
+module.exports = {
+	mergeBrewText : (brew, { style, metadata, fullMetadata })=>{
+		let text = brew.text;
+		if(style) {
+			text = `\`\`\`css\n` +
+                `${brew.style || ''}\n` +
+                `\`\`\`\n\n` +
+                `${text}`;
+		}
+		if(metadata) {
+			let metadata = {
+				title       : brew.title,
+				description : brew.description,
+				tags        : brew.tags,
+				systems     : brew.systems,
+				renderer    : brew.renderer,
+				authors     : brew.authors,
+				published   : brew.published
```

Comment:
Let's also remove `authors` from this. I don't want to have some accidental backdoor for people to manually add other people as authors to brews by just typing into this field. Or accidentally deleting themselves as an author and "losing" their brew somehow.

Reply:
Ah good call -- authors can be added to the `fullMetadata` section below but I need to add a section to `splitTextStyleAndMetadata` that deletes the values that shouldn't be set on the brew from loaded documents. Something like 

```javascript
if(brew.text.startsWith('```metadata')) {
  const index = brew.text.indexOf('```\n\n');
  const metadata = brew.text.slice(12, index - 1);

  const deletionList = ['shareId', 'pageCount', 'authors', 'views', ...];
  for(let key of deletionList) {
    delete metadata[key];
  }

  Object.assign(brew, yaml.load(metadata));
  brew.text = brew.text.slice(index + 5);
}
```
That should prevent anything we don't want set on the brew from being set when the brew is loaded from an external source.